plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/frontend.ts

```diff
@@ -205,11 +226,11 @@ const productFiltersStore = store( 'woocommerce/product-filters', {
 				return ( window.location.href = url.href );
 			}
 
-			const { actions } = yield import(
+			const { actions: routerActions } = yield import(
```

Comment:
I'm not sure we can get better types from generators but this would give the types and not result in type error. Still not ideal:

```ts
const routerModule: typeof import('@wordpress/interactivity-router') =
				yield import( '@wordpress/interactivity-router' );

yield routerModule.actions.navigate( url.href );
```

Reply:
Thanks for the tip! I applied it in 7ed4b86970c5.